mysite/app/templates/app/artist.html

```diff
@@ -0,0 +1,33 @@
+{% extends '../base.html' %}
+
+{% load static %}
+{% block styles %}
+    <link rel="stylesheet" href="{% static 'css/base.css' %}" type="text/css">
+{% endblock %}
+
+{% block content %}
+<div class="page-container">
+    <h1>Konserter som bandet har spilt</h1>
+    <div class="page-content">
+        <table>
+            <thead>
+                <tr>
+                    <th>Tidspunkt</th>
+                    <th>Scene</th>
+                    <th>Tilskuere</th>
```

Comment:
Manager kan godt se profitten, men tilskuere tilhører bookingansvarlig.

Reply:
greit for manager å vite også